live-examples/css-examples/table/border-collapse.html

```diff
@@ -0,0 +1,28 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-collapse">
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_one" class="language-css">border-collapse: collapse;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
```

Comment:
This and line 3 both have `initial-choice="true"` but it doesn't make sense to have it twice :).